docs/infrastructure.md

```diff
@@ -73,11 +73,10 @@ Select account for az:
 $ az account set -s s189-teacher-services-cloud-test
 ```
 
-Get access credentials for a managed Kubernetes cluster (passing the
-resource group and the name):
+Get access credentials for a managed Kubernetes cluster (passing the afqts env):
```

Comment:
```suggestion
Get access credentials for a managed Kubernetes cluster (in this case for the `development` environment):
```

We don't really use the acronym `afqts` so this could be confusing for developers on the team.

Reply:
updated as suggested